src/quo2/components/inputs/input/style.cljs

```diff
@@ -83,15 +83,15 @@
 
 (defn input
   [colors-by-status small? multiple-lines?]
-  (merge (text/text-style {:size :paragraph-1 :weight :regular})
-         {:flex                1
-          :text-align-vertical :top
-          :padding-right       0
-          :padding-left        (if small? 4 8)
-          :padding-vertical    (if small? 4 8)
-          :color               (:text colors-by-status)}
-         (when-not multiple-lines?
-           {:height (if small? 30 38)})))
+  (let [base-props (assoc (text/text-style {:size :paragraph-1 :weight :regular})
```

Comment:
There was a problem I recently noticed on the input component.

On devices with high densitiy of pixels and small font size set by the user, the text wrote inside this input was not centered, it was due to the `:text-align-vertical` added to all the input variants. Now it's only added when the input is `multiline` as suggested by the [React Native docs](https://reactnative.dev/docs/0.63/textinput#multiline).

Extra info: I used a physical Galaxy Z Fold 4 to notice it, previously I didn't see this problem.